src/overlays/actors/ovl_En_Wallmas/z_en_wallmas.c

```diff
@@ -488,8 +489,8 @@ void EnWallmas_TakePlayer(EnWallmas* this, GlobalContext* globalCtx) {
 
         this->timer += 2;
     } else {
-        Math_StepToF(&this->actor.world.pos.y, sYOffsetPerForm[(void)0, gSaveContext.playerForm] + player->actor.world.pos.y,
-                     5.0f);
+        Math_StepToF(&this->actor.world.pos.y,
+                     sYOffsetPerForm[(void)0, gSaveContext.playerForm] + player->actor.world.pos.y, 5.0f);
```

Comment:
Hmm, I think I'd put brackets round this even when they're not necessary, since we'll almost certainly end up making it a macro.